client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -479,7 +481,7 @@ function StatsCommercialFlowOptOutForm( {
 				</ul>
 			</div>
 			<div className={ `${ COMPONENT_CLASS_NAME }__personal-checklist-button` }>
-				{ supportsOnDemandCommercialClassification && (
+				{ supportsOnDemandCommercialClassification && isCommercial && (
```

Comment:
The button is also the `Continue` button, which we don't want to hide for non-commercial users.